packages/web/src/components/track/types.ts

```diff
@@ -62,6 +62,9 @@ export type TrackTileProps = TileProps & {
   artistHandle: string
   artistIsVerified: boolean
   isFeed?: boolean
+  isDM?: boolean
```

Comment:
I think this is better called `hideBottomBar` or `hideUserActions` rather than coupling it with the one usage for DMs?

Reply:
sounds good. we do use the isDM logic elsewhere around here. but i guess can have on more prop to be explicit

Reply:
hmm wdyt? maybe. feel like the additional prop would then be redundant

Reply:
Ah I didn't realize it was also in use for analytics. Either case this is super minor, I'm fine leaving it as is and refactoring later if needed for something else

Reply:
isChat it is